inc/Engine/Admin/RocketInsights/Render.php

```diff
@@ -306,4 +316,49 @@ public function truncate_title( string $title ): array {
 			'is_truncated'    => true,
 		];
 	}
+
+	/**
+	 * Renders the Rocket Insights column content for post listing pages.
+	 *
+	 * @since 3.20.1
+	 *
+	 * @param string $url The URL of the post.
+	 *
+	 * @return void
+	 */
+	public function render_rocket_insights_column( string $url ): void {
```

Comment:
My suggestion: change this to be `get_rocket_insights_column()`, and `render_rocket_insights_column()` is only a wrapper for `echo get_rocket_insights_column()`